moveit_ros/hybrid_planning/local_planner/local_planner_component/src/local_planner_component.cpp

```diff
@@ -96,6 +96,12 @@ bool LocalPlannerComponent::initialize()
   planning_scene_monitor_->startWorldGeometryMonitor(config_.collision_object_topic);
   planning_scene_monitor_->startStateMonitor(config_.joint_states_topic);
 
+  // Create a copy of the planning scene
+  planning_scene_monitor_->lockSceneRead();  // LOCK planning scene
+  planning_scene::PlanningScenePtr planning_scene = planning_scene_monitor_->getPlanningScene();
+  planning_scene->decoupleParent();
```

Comment:
The planning_scene_monitor already maintains a decoupled scene I guess.
You are trying to decouple the planning scene that is modified inside the monitor here.
Why? I expect that these lines are a no-op.